src/app/static/src/app/components/header/OnlineSupportMenu.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <drop-down text="support" :right="true" style="flex: none">
+        <a class="dropdown-item"
+           :href="'https://mrc-ide.github.io/naomi-troubleshooting/' + troubleFilename"
```

Comment:
Not sure why you've split up knowledge of the troubleshooting document between this component and its container? There is only one - I'd have though the url could just live inside this component, as the bug report ones do.